### PR TITLE
Project specific CD User

### PR DESCRIPTION
**Pull request by tjaeschke**
merge base

### CODE REVIEW

**Comment by tjaeschke** [comment]
relates to https://github.com/opendevstack/ods-project-quickstarters/issues/289

**Comment by tjaeschke** [comment]
Would you please check the implementation?

File: ocp-config/prov-app/cm.yml [hunk 1]
```diff
@@ -63,6 +63,9 @@ objects:
       bitbucket.default.user.group=${PROV_APP_CROWD_USER_GROUP}
       bitbucket.technical.user=${CD_USER_ID}
 
+      scm.global.readablerepos.opendevstack[0]=ods-jenkins-shared-library
+      scm.global.readablerepos.opendevstack[1]=ods-ods-quickstarters
```
Comment (michaelsauter) [inline ocp-config/prov-app/cm.yml]: Typo? Should be just `ods-quickstarters`, right? Also, we need to read the MRO library. I'd rather work based on exclusion if possible?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -82,6 +82,10 @@ bitbucket.repository.pattern=%s-%s
 bitbucket.default.user.group=opendevstack-administrators
 bitbucket.technical.user=cd_user
 
+
+scm.global.readablerepos.opendevstack[0]=ods-jenkins-shared-library
+scm.global.readablerepos.opendevstack[1]=ods-ods-quickstarters
```
Comment (michaelsauter) [inline src/main/resources/application.properties]: Same typo here ...

**Comment by michaelsauter** [comment]
@stefanlack Can you please share how this affects the API? I think there must be a way to pass the technical user name  into it but I fail to see where this happens here. Do you know?

File: ocp-config/prov-app/cm.yml (hunk 1 above)
Comment (stefanlack) [reply ocp-config/prov-app/cm.yml]: Done in #297

File: src/main/resources/application.properties (hunk 2 above)
Comment (stefanlack) [reply src/main/resources/application.properties]: Done in #297

**Comment by stefanlack** [comment]
Closing this pr. It will be replaced by #297

**Pull request closed without merging**